ui/lib/kubernetes/addon/components/page/overview.hbs

```diff
@@ -3,32 +3,27 @@
 </TabPageHeader>
 
 {{#if @config}}
-  <div class="has-top-margin-m">
-    <div class="is-flex">
-      <div class="box has-padding-m has-right-margin-m is-rounded column is-flex-half" data-test-roles-card>
-        <div class="is-flex-between">
-          <h2 class="title is-3">
-            Roles
-          </h2>
-          {{#if @roles.length}}
-            <LinkTo class="is-no-underline" @route="roles">View Roles</LinkTo>
-          {{else}}
-            <LinkTo class="is-no-underline" @route="roles.create">Create Role</LinkTo>
-          {{/if}}
-        </div>
-        <p>The number of Vault roles being used to generate Kubernetes credentials.</p>
-        <h2 class="title has-font-weight-normal is-3 has-top-margin-l">
-          {{or @roles.length "None"}}
-        </h2>
-      </div>
-      <div class="box has-padding-m is-rounded column is-flex-half" data-test-generate-credential-card>
-        <h2 class="title is-3">
-          Generate credentials
-        </h2>
-        <p>Quickly generate credentials by typing the role name.</p>
-        <div class="is-flex-center has-top-margin-s">
+  <div class="selectable-card-container has-grid has-top-margin-l has-two-col-grid">
+    <OverviewCard
+      @cardTitle="Roles"
+      @subText="The number of Vault roles being used to generate Kubernetes credentials."
+      @actionText={{if @roles.length "View Roles" "Create Role"}}
+      @actionTo={{if @roles.length "roles" "roles.create"}}
+    >
+      <h2 class="title is-2 has-font-weight-normal has-top-margin-m" data-test-roles-card-overview-num>{{or
+          @roles.length
+          "None"
+        }}</h2>
+    </OverviewCard>
+    <OverviewCard @cardTitle="Generate credentials" @subText="Quickly generate credentials by typing the role name.">
+      <form
+        aria-label="generate credentials"
+        {{on "submit" this.generateCredential}}
+        data-test-selectable-card="Generate credentials"
+      >
```

Comment:
I'm wondering if we should stick with the action on the button instead of adding this form since we aren't actually creating or submitting anything here, only transitioning to a different page?

Reply:
I'm not sure what best practice is here in terms of accessibility. Since it has an input and button, I thought of it as a form but it does make sense that this just transitions to a different page.  Claire also mentioned for pki overview so I will update!